libcxx/test/std/ranges/range.factories/range.iota.view/empty.pass.cpp

```diff
@@ -0,0 +1,157 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17
+
+// constexpr bool empty() const;
+
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <utility>
+#include <vector>
+
+#include "types.h"
+
+template <typename R>
+concept HasEmpty = requires(R r) {
```

Comment:
```suggestion
concept HasEmpty = requires(R const r) {
```

Otherwise you don't test anywhere that the method is marked as `const`.

Reply:
Done